core/tauri-build/src/allowlist.rs

```diff
@@ -43,7 +43,12 @@ pub fn check(config: &Config, manifest: &mut Manifest) -> Result<()> {
       name: "tauri".into(),
       alias: None,
       kind: DependencyKind::Normal,
-      all_cli_managed_features: Some(TauriConfig::all_features()),
+      all_cli_managed_features: Some(
+        TauriConfig::all_features()
+          .into_iter()
+          .filter(|f| f != &"tray-icon")
+          .collect(),
+      ),
```

Comment:
should it be `TauriConfig::default_features()`? @lucasfernog

Reply:
not really, this just checks if the CLI managed features are correct on Cargo.toml, but tray-icon might be enabled by the user while the CLI thinks it shouldn't be (because the config entry is not set).